docs/integrations/sources/salesloft.md

```diff
@@ -91,6 +97,7 @@ Salesloft has the [rate limits](hhttps://developers.salesloft.com/api.html#!/Top
 
 | Version | Date       | Pull Request                                             | Subject                                                           |
 |:--------|:-----------|:---------------------------------------------------------|:------------------------------------------------------------------|
+| 1.1.2   | 2023-06-20 | [27505](https://github.com/airbytehq/airbyte/pull/27505) | Added new streams (Call Data Records, Call Dispositions, ... )    |
```

Comment:
Can you bump this to 1.2.0? Based on semantic versioning here: https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors